CHANGELOG.md

```diff
@@ -3,14 +3,10 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## Unreleased
 
-### Added
-- 
-
```

Comment:
To clarify: As part of the pre-publishing process, Travis takes everything under the "Unreleased" heading and adds a version number and timestamp above it.